Jazzy issue while compiling a mapbox source with XCode 8

**Platform:** OSX 10.11.5, XCode 8.0 (8A218a)
**Mapbox SDK version:** current master branch
I met an error while building a project with the 'make iframework' command:

```
* Generating API documentation…
OUTPUT=build/ios/pkg/documentation ./platform/ios/scripts/document.sh
Using config file /Users/mike/Documents/ios_projects/mapbox-gl-native/platform/ios/jazzy.yml
xcrun: error: unable to find utility "xcrun", not a developer tool or in PATH
/Users/mike/.rbenv/versions/2.3.1/lib/ruby/2.3.0/open3.rb:199:in `spawn': No such file or directory -  (Errno::ENOENT)
	from /Users/mike/.rbenv/versions/2.3.1/lib/ruby/2.3.0/open3.rb:199:in `popen_run'
	from /Users/mike/.rbenv/versions/2.3.1/lib/ruby/2.3.0/open3.rb:95:in `popen3'
	from /Users/mike/.rbenv/versions/2.3.1/lib/ruby/2.3.0/open3.rb:258:in `capture3'
	from /Users/mike/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/xcinvoke-0.2.1/lib/xcinvoke/xcode.rb:86:in `xcrun'
	from /Users/mike/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/xcinvoke-0.2.1/lib/xcinvoke/xcode.rb:119:in `swift_info'
	from /Users/mike/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/xcinvoke-0.2.1/lib/xcinvoke/xcode.rb:57:in `swift_version'
	from /Users/mike/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/xcinvoke-0.2.1/lib/xcinvoke/xcode.rb:53:in `block in find_swift_version'
	from /Users/mike/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/xcinvoke-0.2.1/lib/xcinvoke/xcode.rb:44:in `each'
	from /Users/mike/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/xcinvoke-0.2.1/lib/xcinvoke/xcode.rb:44:in `each'
	from /Users/mike/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/xcinvoke-0.2.1/lib/xcinvoke/xcode.rb:53:in `select'
	from /Users/mike/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/xcinvoke-0.2.1/lib/xcinvoke/xcode.rb:53:in `find_swift_version'
	from /Users/mike/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/jazzy-0.7.2/lib/jazzy/sourcekitten.rb:139:in `run_sourcekitten'
	from /Users/mike/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/jazzy-0.7.2/lib/jazzy/doc_builder.rb:64:in `block in build'
	from /Users/mike/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/jazzy-0.7.2/lib/jazzy/doc_builder.rb:62:in `chdir'
	from /Users/mike/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/jazzy-0.7.2/lib/jazzy/doc_builder.rb:62:in `build'
	from /Users/mike/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/jazzy-0.7.2/bin/jazzy:15:in `<top (required)>'
	from /Users/mike/.rbenv/versions/2.3.1/bin/jazzy:23:in `load'
	from /Users/mike/.rbenv/versions/2.3.1/bin/jazzy:23:in `<main>'
make[1]: *** [idocument] Error 1
make: *** [iframework] Error 2
```

I've used a temporal solution from here https://github.com/realm/jazzy/issues/656 with success.

--- a/platform/ios/scripts/document.sh
+++ b/platform/ios/scripts/document.sh
@@ -19,8 +19,6 @@ BRANCH=$( git describe --tags --match=ios-v*.*.* --abbrev=0 )
 SHORT_VERSION=$( echo ${BRANCH} | sed 's/^ios-v//' )
 RELEASE_VERSION=$( echo ${SHORT_VERSION} | sed -e 's/^ios-v//' -e 's/-.*//' )
 
-SWIFT_VERSION=$(xcrun swift -version | head -n 1 | sed -e 's/^Apple Swift version //' -e 's/ .*$//')
-
 rm -rf /tmp/mbgl
 mkdir -p /tmp/mbgl/
 README=/tmp/mbgl/README.md
@@ -37,7 +35,6 @@ cp platform/ios/screenshot.png "${OUTPUT}"
 jazzy \
     --config platform/ios/jazzy.yml \
     --sdk iphonesimulator \
-    --swift-version $SWIFT_VERSION \
     --github-file-prefix https://github.com/mapbox/mapbox-gl-native/tree/${BRANCH} \
     --module-version ${SHORT_VERSION} \
     --readme ${README} \

